resources/views/components/user/profile/primary-meta.blade.php

```diff
@@ -101,4 +101,25 @@ class="rounded-sm h-[64px] w-[64px] sm:max-h-[128px] sm:max-w-[128px] sm:min-w-[
             <x-user.profile.follows-you-label :username="$username" />
         </div>
     </div>
+
+    @if ($amIModerator)
+        <button class="absolute bottom-0 right-0 btn" onclick="toggleModeratorTools()">
+            Moderator ▼
```

Comment:
In reference to maybe linking to Filament admin panels in the future, I had those initially labeled in verbs, like in this case it would've been "Moderate". Does this apply here, what do you think?

Reply:
No issues here on changing this, updated to "Moderate" in latest.